src/index.html

```diff
@@ -8,6 +8,73 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
```

Comment:
no needs to add extra line between _oneline_ siblings 
```suggestion
        <span class="stars__star"></span>
        <span class="stars__star"></span>
```